move/sources/aptogotchi.move

```diff
@@ -49,9 +49,14 @@ module aptogotchi::main {
         id: u64,
     }
 
-    // Tokens require a signer to create, so this is the signer for the collection
-    struct AppCapability has key {
-        extend_ref: ExtendRef,
+    // We need a contract signer as the creator of the aptogotchi collection and aptogotchi token
+    // Otherwise we need admin to sign whenever a new aptogotchi token is minted which is inconvenient
+    struct Config has key {
```

Comment:
is Config a convention way to name this? I feel like AppCapability at least gives us some idea what it's about. but config doesn't mean anything

Reply:
Let me ask around, https://github.com/aptos-labs/aptos-developer-discussions/discussions/69